Loop/Public/FeedbackLoop.swift

```diff
@@ -144,8 +184,30 @@ extension Loop {
             skippingRepeated transform: @escaping (State) -> Control?,
             effects: @escaping (Control) -> Effect
         ) where Effect.Value == Event, Effect.Error == Never {
-            self.init(compacting: { $0.map(transform).skipRepeats() },
-                      effects: { $0.map(effects)?.producer ?? .empty })
+            self.init(
+                compacting: { $0.map(transform).skipRepeats() },
+                effects: { $0.map(effects)?.producer ?? .empty }
+            )
+        }
+
+        public static func skippingRepeated<Control: Equatable, Effect: SignalProducerConvertible>(
```

Comment:
Per your comment [here](https://github.com/babylonhealth/ReactiveFeedback/pull/59#discussion_r414721583) @andersio , maybe we just support both for now, so people can use what ever they prefer>

Reply:
If that's the case, I'd prefer not to duplicate the implementation. We should implement one of them in terms of another.